syz-cluster/README.md

```diff
@@ -2,8 +2,12 @@
 
 1. Install and start minikube: https://minikube.sigs.k8s.io/docs/start/
 ```
-$ minikube start
+$ minikube start --cni=cilium
 ```
+
+`--cni=cilium` enables the use of a more advanced Network plugin that supports
+the emulation of network policies.
```

Comment:
Why is it needed?

Reply:
By default, `minikube` does not emulate network policies.